.gitlab-ci.yml

```diff
@@ -375,33 +369,11 @@ variables:
 # Workflow rules
 # Rules used to define whether a pipeline should run, and with which variables
 #
-
-# WARNING: Do not change below if you want to globally disable the caching proxy (all branches would need to be rebased for it to be effectively disabled).
-# Instead use the break-glass mechanism by looking for the "Remotely disable Artifactory" doc in Confluence.
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❓ question:** ‏ do we still need/have a break-glass mechanism to disable using the new caching solution?

Reply:
I don't think anything like that would be applicable to the pieces that are partially replacing the role of the caching mirror.